qemu/tests/rng_hotplug.py

```diff
@@ -122,28 +130,36 @@ def run_subtest(sub_test):
                                        {'id': 'virtio-rng-pci-%d' % num})
             hotplug_rng(vm, new_dev)
             dev_list.append(new_dev)
+            # temporary workaround for migration
+            vm.params["virtio_rngs"] += ' rng%d' % num
+            vm.params["backend_rng%d" % num] = 'rng-random'
```

Comment:
I can't see the test restricted other rng types, so should not make this hardcoding.